webxrlayers-1.bs

```diff
@@ -198,6 +201,12 @@ navigator.xr.requestSession('immersive-vr', {
 Layers are only supported for XRSessions created with XRSessionMode of {{XRSessionMode/"immersive-vr"}}
 or {{XRSessionMode/"immersive-ar"}}. {{XRSessionMode/"inline"}} sessions MUST not support layers.
 
+If a session is created without "[=feature descriptor/layers=]" support, it MUST NOT be granted the
+"[=feature descriptor/layers=]" feature using the [=request the xr permission=] algorithm.
```

Comment:
I don't like calling out this argument in particular, can we instead just mention that the layer descriptor has a [feature requirement](https://immersive-web.github.io/webxr/#feature-requirements) that it must not be called when there is an active XRSession of the requested mode?

Reply:
> I don't like calling out this algorithm in particular, can we instead just mention that the layer descriptor has a [feature requirement](https://immersive-web.github.io/webxr/#feature-requirements) that it must not be enabled when there is an active XRSession of the requested mode, with a note saying that this cannot be enabled mid-session.

How would we specify that? The link you provided list requirement, not features that should not be enabled.

> Though with the current approach you have the ability to request this feature and let it get enabled the next session.

Yes, that's the way I wrote it since you alluded that this is possible. 
However, if you request a feature (using `request` API), will it be enabled next time you call requestSession (even if you don't enable layers)?

I think that would be really strange, and also break inline sessions since they can't have layers.

Reply:
> How would we specify that? The link you provided list requirement, not features that should not be enabled.

sure, but the requirement could be "not in the middle of a session"

> and also break inline sessions since they can't have layers.

ah, then we can just do it for immersive sessions. it wouldn't break inline sessions though.

>  However, if you request a feature (using request API), will it be enabled next time you call requestSession (even if you don't enable layers)?

no, but it may not trigger a consent prompt this time.

the use case at hand is an application deciding later on to request layers, but that doesn't seem super useful: you should just have it in optionalFeatures initially and check later

Reply:
@Manishearth I don't quite know how to phrase this.
Can you make a suggestion?

Reply:
Done

Reply:
Others might disagree with this approach, though :smile:, worth waiting to see what Brandon or Mounir think

Reply:
I think this is a reasonable approach. I will merge it but keep the issue open for now.